src/iop/ashift.c

```diff
@@ -5698,7 +5700,7 @@ void gui_init(struct dt_iop_module_t *self)
 
   self->widget = main_box;
 
-  gtk_widget_set_tooltip_text(g->rotation, _("rotate image"));
+  gtk_widget_set_tooltip_text(g->rotation, _("right-click and drag a line on the image to drag a straight line"));
```

Comment:
Should still reference the functionality of the slider, plus slight wording change. How about...

```suggestion
  gtk_widget_set_tooltip_text(g->rotation, _("rotate image\nright-click and drag to define a horizontal or vertical line by drawing on the image"));
```